dockers/docker-sonic-restapi/supervisord.conf

```diff
@@ -13,7 +13,7 @@ events=PROCESS_STATE
 buffer_size=1024
 
 [eventlistener:supervisor-proc-exit-listener]
-command=/usr/bin/supervisor-proc-exit-listener --container-name restapi
+command=python /usr/bin/supervisor-proc-exit-listener --container-name restapi
```

Comment:
Suggest replacing `python` with `python2` so that is is more explicit and obvious what we are doing here, in case issues arise, until the container can be upgraded to Buster w/Python 3.

Also, have you confirmed that `supervisor-proc-exit-listener` is still backward-compatible with Python 2?

Reply:
Good suggestion. Updated. Thanks
I've confirmed that ```supervisor-proc-exit-listener``` is still compatible with ```Python 2```. I verified that by static check and dynamic running.